manifests/w/WebTorrent/WebTorrent/0.24.0/WebTorrent.WebTorrent.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WebTorrent.WebTorrent
+PackageVersion: 0.24.0
+PackageLocale: en-US
+Publisher: WebTorrent, LLC
+PackageName: WebTorrent
+License: MIT
+Copyright: Copyright © 2020 WebTorrent, LLC
+ShortDescription: WebTorrent

```

Comment:
Please provide a better description of what this package is, does, or is used for